matching-notification-api/src/main/java/org/phenotips/matchingnotification/match/internal/DefaultPatientInMatch.java

```diff
@@ -26,9 +26,11 @@
 import org.phenotips.data.permissions.AccessLevel;
 import org.phenotips.data.permissions.EntityPermissionsManager;
 import org.phenotips.data.permissions.internal.EntityAccessManager;
+import org.phenotips.data.similarity.AccessType;
 import org.phenotips.data.similarity.PatientGenotype;
 import org.phenotips.data.similarity.PatientGenotypeManager;
 import org.phenotips.data.similarity.PatientSimilarityView;
+import org.phenotips.data.similarity.internal.DefaultAccessType;
```

Comment:
We can check view access to a `Patient` without using classes from other internal package - not sure why similarity uses those (I think because there it deals with two patients, - but we only have on here). Here we have standard access managers available which have nothing to do with similarity and which can tell if current user has view access. In fact I think `this.access` already has all you need directly, and use it e.g. on line 221 below

Reply:
If we are aiming patient match page to be consistent with similar matches, we need to use the same methods to determine the access to the patient external ID. RestrictedSimilarityView uses DefaultAccessType for that purpose. If we here will use different method it is prone to future bugs in case DefaultAccessType will change and we will have 2 different methods.